charts/traces/Chart.lock

```diff
@@ -1,12 +1,12 @@
 dependencies:
 - name: opentelemetry-collector
   repository: https://open-telemetry.github.io/opentelemetry-helm-charts
-  version: 0.61.2
+  version: 0.69.0
```

Comment:
anything higher than `0.69.0` doesn't pass the tests. Not sure if a breaking change was introduced or just our tests aren't compatible. Leaving it as an exercise for future us

[relevant git diff](https://github.com/open-telemetry/opentelemetry-helm-charts/compare/opentelemetry-collector-0.69.0...opentelemetry-collector-0.69.1)